backend/api/v1/endpoints/auth.py

```
@@ -36,11 +47,19 @@
         username=user.username,
         email=user.email,
         hashed_password=hashed_password,
-        profile_completed=False  # ✅ Profile completion starts separately
+        profile_completed=False,  # ✅ Profile completion starts separately
+        is_verified=False  # ✅ Email verification status
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)